CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ if (LLVM_DG)
 
 	add_definitions(${LLVM_DEFINITIONS})
 
-	if (${LLVM_PACKAGE_VERSION} VERSION_GREATER "3.4")
+	if (NOT ${LLVM_PACKAGE_VERSION} VERSION_LESS "3.5")
 		llvm_map_components_to_libnames(llvm_libs support core
```

Comment:
Perhaps `llvm_map_components_to_libraries` should be used when `if (${LLVM_PACKAGE_VERSION} VERSION_LESS "3.5")`, and `llvm_map_components_to_libnames` otherwise?  The components names work on both versions.

At least that's what I thought made sense previously, to be honest not sure if that's needed/useful; omitting it without an equivalent might cause some underlinking problems in certain build configurations.  Maybe. :).